docs/integration_resources/resources/usd_tutorials/usd_tutorial_02.md

```diff
@@ -0,0 +1,162 @@
+## Inspecting and Authoring Properties
+
+This tutorial is focused on looking at the properties containing geometric data on the prims created in the [Creating Your First USD Stage](usd_tutorial_01.md) tutorial, using the operators from the Shift_USD catalog.
+
+Also, the base file for the example can be downloaded from the [OpenUSD](https://github.com/PixarAnimationStudios/OpenUSD) Github project repository, from `extras/usd/tutorials/authoringProperties/` folder, it is the `HelloWorld.usda` file.
+
+In order to follow easily the steps, it is recommended to read the documentation about the [Shift's interface](../../../getting_started/basics/ui_overview.md) and [workflow execution](../../../getting_started/basics/execute.md).
+
+
+### Tutorial
+
+1. Open Shift application. A new window's instance of Shift should appear like the following one:
+
+<figure>
+    <img src="images/usd_tutorial_02/step_01_t02.png" alt="Shift UI.">
+    <figcaption><b>Figure 1</b>: Shift UI.</figcaption>
+</figure>
+
+2. Now, add a new **<tt>UsdStageOpen</tt>** node to the board. Using the **Inspector** widget, set as value for the **filepath** input plug a filepath pointing to the `HelloWorld.usda` file downloaded for the example. For doing this, the UI offers a browse button next to the plug input field that opens a browse dialog to facilitate locating the file or it can be added typing manually the path. The board should look like the following one:
```

Comment:
We are not using `<tt>` in the first tutorial. Why do we here??
Also, I don't think is used in the rest of the documentation. Usually, we use quotes or we set it to italic.

If you still want to use this format, please ensure that is consistent through all the tutorials. 
```suggestion
2. Now, add a new **<tt>UsdStageOpen</tt>** node to the board. Using the **Inspector** widget, set as value for the **filepath** input plug a filepath pointing to the `HelloWorld.usda` file downloaded for the example. For doing this, the UI offers a browse button next to the plug input field that opens a browse dialog to facilitate locating the file or it can be added typing manually the path. The board should look like the following one:
```

Reply:
In the first tutorial we're using `**<tt>{node_name}</tt>**` too. I did it in this way because it's easier to mention the name of the node to be created. Then I use `{node_name}` to refer to the name of the node in the board after renaming it.